gravitee-plugin-annotation-processors/src/test/resources/test/TestConfigurationEvaluator.java

```diff
@@ -257,6 +267,7 @@ public Single<TestConfiguration> eval(ExecutionContext ctx) {
         // Evaluate properties that needs EL, validate evaluatedConf and returns it
         return Maybe
             .merge(Flowable.fromIterable(toEval))
+            .doOnError(throwable -> logger.error("Exception thrown before validation", throwable))
```

Comment:
Probably better to not log the exception as it could flood the logs.
The other option could be to move it into the evalStringProperty so it is possible to log a message with enough information, ex:
```java
        //Then check EL
        return ctx.getTemplateEngine().eval(value, String.class)
                .doOnError(throwable -> logger.error("Unable to evaluate property [{}] with expression [{}].", name, value));
```

Reply:
Yep I'll change that 👍